test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java

```diff
@@ -67,6 +68,14 @@ public void testFailure(Failure failure) throws Exception {
         String task = System.getProperty("tests.task");
         boolean isBwcTest = Boolean.parseBoolean(System.getProperty("tests.bwc", "false"));
 
+        for (var bwcProperty : List.of("tests.bwc.main.version", "tests.bwc.refspec.main")) {
```

Comment:
There's already a mechanism here for appending properties, see `GradleMessageBuilder.appendESProperties()`